### PR TITLE
Implement sorting logic

### PR DESCRIPTION
**Pull request by alexCatus**

### CODE REVIEW

**Comment by JadSafah** [comment]
For some reason, only one of the 3 icons are showing up as an arrow for me 🤔 

Screen Shot 2022-01-13 at 11 57 43 AM

File: apps/frontend/src/app/app.component.scss [hunk 1]
```diff
@@ -42,4 +42,27 @@ thead {
...
+
+button {
+  border:none;
+  border-radius: 50%;
+  padding: 10px;
+  background: no-repeat left center;
+  background-size: cover;
+}
+.btn-basic {
+  background-image:url('../assets/minus-svgrepo-com.svg') ;
```
Comment (eharason) [inline apps/frontend/src/app/app.component.scss:55]: @JadSafah Alex did that on purpose! btn-pasic is that minus sign lol. it is KINDA odd, but everything's working ok! do we have other images perhaps? 🤔

Comment (eharason) [reply apps/frontend/src/app/app.component.scss:55]: maybe we could do this one?
https://icons8.com/icons/set/sort

**Review by eharason — APPROVED** [review]
it works well! we can always open a new ticket to change the minus button to a sorting one e.g. 
image

File: apps/frontend/src/app/app.component.scss (hunk 1 above)
Comment (JadSafah) [reply apps/frontend/src/app/app.component.scss:55]: oh ok! It looked too clean to be a mistake lol

**Review by JadSafah — APPROVED** [review]

File: apps/frontend/src/app/app.component.scss (hunk 1 above)
Comment (alexCatus) [reply apps/frontend/src/app/app.component.scss:55]: i LOVE this one emily! I wanted your input for the icons anyway :D :D :D :D

File: apps/frontend/src/app/app.component.ts [hunk 2]
```diff
@@ -1,17 +1,37 @@
...
+        }))
+
 
-  constructor(private songService: SongsService) { }
+  constructor(private songService: SongsService) {
+  }
 
-  getSongs() {
-    return this.songService.songs;
+  newDirection(selectedColumn: SortUtils.Column) {
```
Comment (alexCatus) [inline apps/frontend/src/app/app.component.ts:32]: nobody saw what i did here i'm so disappointed @eharason @JadSafah @atbriggs

**Pull request merged**